benchark/get_json_object_stress_gen.scala

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+spark.conf.set("spark.rapids.sql.enabled", false)
+
+import org.apache.spark.sql.tests.datagen._
+import org.apache.spark.sql.types._
+
+val numRows = 3000000
+//val nullProbability = 0.1
+val nullProbability = 0.0001
+val output = "/data/tmp/SCALE_FROM_JSON"
```

Comment:
both input and output assume there's a `/data/` folder. That may be fine  and can be improved upon later, but perhaps a note would be good?